tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -105,29 +105,27 @@ namespace tuplex {
         assert(stage);
 
         // reset history server
-        _historyServer.reset();
+//        _historyServer.reset();
```

Comment:
can this get removed? why is the resetting deactivated?

Reply:
The resetting logic was incorrect. Since the execute is recursive, the history server would be reset at the beginning of each stage. I changed it to be reset only at the beginning of new jobs.